cli/gh-pr-filters.go

```diff
@@ -89,14 +89,15 @@ func GetFilterForMilestone(milestoneRaw string) *Filter {
 		PR: func(pr github.PullRequest) (bool, error) {
 			milestone := pr.GetMilestone().GetTitle()
 
-			// nolint:gocritic
+			//nolint:gocritic
 			if strings.EqualFold(filterMilestone, milestone) && !negate {
 				c.Printf("    milestone: <green>%s</> <gray>(%s)</>\n", filterMilestone, milestone)
 				return true, nil
 			} else if negate {
 				c.Printf("    milestone: <green>-%s</> <gray>(%s)</>\n", filterMilestone, milestone)
 				return true, nil
 			} else {
+				//revive:disable:indent-error-flow
 				c.Printf("    milestone: <red>%s</> <gray>(%s)</>\n", filterMilestone, milestone)
 				return false, nil
 			}
```

Comment:
cli/gh-pr-filters.go:99:11: indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)
```
                        } else {
                                c.Printf("    milestone: <red>%s</> <gray>(%s)</>\n", filterMilestone, milestone)
                                return false, nil
                        }
```